makefile

```diff
@@ -98,7 +101,10 @@ EXOBJS	=\
 	$(oDir)/ST_stats.o\
 	$(oDir)/sxw_environs.o\
 	$(oDir)/ST_grid.o\
-	$(oDir)/ST_sql.o
+	$(oDir)/ST_sql.o\
+	$(oDir)/ST_initialization.o\
+	$(oDir)/ST_progressBar.o\
+	$(oDir)/ST_seedDispersal.o
 	#$(oDir)/sxw_tester.o
```

Comment:
Can we get rid of this commented-out object file? If we need it again, we can add it back later.

Reply:
Resolved by 9b99edd.